tests/e2e/helper_test.go

```diff
@@ -168,9 +166,12 @@ func setupDSCInstance(name string) *dscv1.DataScienceCluster {
 						ManagementState: operatorv1.Removed,
 					},
 				},
-				ModelRegistry: modelregistry.ModelRegistry{
-					Component: componentsold.Component{
-						ManagementState: operatorv1.Removed,
+				ModelRegistry: componentsv1.DSCModelRegistry{
+					ManagementSpec: components.ManagementSpec{
+						ManagementState: operatorv1.Managed,
+					},
+					ModelRegistryCommonSpec: componentsv1.ModelRegistryCommonSpec{
+						RegistriesNamespace: "odh-model-registries",
```

Comment:
maybe use DefaultModelRegistriesNamespace, so it will work with RHOAI e2e later

Reply:
done